L/libcap/build_tarballs.jl

```diff
@@ -18,15 +18,9 @@ sources = [
 script = raw"""
 cd $WORKSPACE/srcdir/libcap-*/
 
-# GNU (not musl) 64-bit archs put their loader in `/lib64`
-if [[ ${target} == *gnu* ]] && [[ ${nbits} == 64 ]]; then
-    LOADER_DIR=lib64
-else
-    LOADER_DIR=lib
-fi
-
 make -j${nproc} BUILD_CC=${BUILD_CC} COPTS="-O2 -std=c99"
-make install DESTDIR=${prefix} prefix=/ lib=${LOADER_DIR} RAISE_SETFCAP=no
+make install DESTDIR=${prefix} prefix=/ lib=lib
+error
```

Comment:
Do you know about [`--debug=end`](https://docs.binarybuilder.org/stable/reference/#Command-Line)? :wink:

Reply:
Oh my god no I didn't...  🤦  Thanks for the tip!